### PR TITLE
DOC/CLN: Fix errors in DataFrame docstrings

### PR DESCRIPTION
**Pull request by dsaxton**
A few small edits to `DataFrame` docstrings

### CODE REVIEW

File: pandas/core/frame.py [hunk 1]
```diff
@@ -2739,7 +2739,7 @@ def get_value(self, index, col, takeable=False):
 
         Returns
         -------
-        value : scalar value
+        scalar value
```
Comment (WillAyd) [inline pandas/core/frame.py]: Would be preferable to state the type of scalar being returned here

File: pandas/core/frame.py [hunk 2]
```diff
@@ -3569,7 +3569,7 @@ def _sanitize_column(self, key, value, broadcast=True):
 
         Returns
         -------
-        sanitized_column : numpy-array
+        numpy-array
```
Comment (WillAyd) [inline pandas/core/frame.py]:
```suggestion
        numpy.ndarray
```

File: pandas/core/frame.py [hunk 3]
```diff
@@ -5032,7 +5032,7 @@ def swaplevel(self, i=-2, j=-1, axis=0):
 
         Returns
         -------
-        swapped : same type as caller (new object)
+        same type as caller (new object)
```
Comment (WillAyd) [inline pandas/core/frame.py]: Hmm I think when we've done this for other objects we've said something like `DataFrame or Series` (or whatever is applicable) and left `Same type as caller` as a comment

File: pandas/core/frame.py [hunk 4]
```diff
@@ -5959,7 +5959,7 @@ def unstack(self, level=-1, fill_value=None):
 
         Returns
         -------
-        unstacked : DataFrame or Series
+        DataFrame or Series
```
Comment (WillAyd) [inline pandas/core/frame.py]: Super nitpicky but would be nice to order these the same way whenever used. A quick grep on codebase tells me `Series or DataFrame` appears more often so maybe stick with that ordering

@datapythonista something to consider

**Review by WillAyd — CHANGES_REQUESTED** [review]

File: pandas/core/frame.py (hunk 1 above)
Comment (WillAyd) [reply pandas/core/frame.py]: Good question. What you've suggested seems like the most logical thing to me!

cc @datapythonista

File: pandas/core/frame.py (hunk 3 above)
Comment (dsaxton) [reply pandas/core/frame.py]: I switched this to `DataFrame`, since that should always be the type of the caller here

**Comment by WillAyd** [comment]
Can you merge master?

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @dsaxton !

**Pull request merged**